client/Components/Dogs/Dogs.jsx

```diff
@@ -1,20 +1,41 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { Link } from 'react-router-dom';
 
+import FilterSideBar from './../FilterSideBar';
 import { ROUTE_PATH } from './../../commons/constants';
 const { DOGS } = ROUTE_PATH;
 
-const Dogs = ({ dogs }) => (
-  <div>
-    {dogs.map(dog => (
-      <div key={dog.id}>
-        <Link to={`${DOGS}/${dog.id}`}>{dog.name}</Link> {dog.price}{' '}
-        {dog.description}
+class Dogs extends Component {
+  constructor() {
+    super();
+    this.state = {
+      breedId: 0
+    };
```

Comment:
just aesthetic, but u can take off constructor and just this.state = {}

Reply:
haven't tried this syntax yet...